src/commands/Bunnies.js

```diff
@@ -0,0 +1,48 @@
+const { Command } = require('@dyno.gg/dyno-core');
+const superagent = require('superagent');
+
+
+class Bunnies extends Command {
+    constructor(...args) {
+        super(...args);
+
+        this.aliases      = ['bunny', 'bunnies'];
+        this.module       = 'Fun';
+        this.description  = 'Random Adorable Bunnies';
+        this.usage        = 'bunny';
+        this.example      = 'bunny';
+        this.cooldown     = 7500;
+        this.expectedArgs = 0;
+    }
+
+    async execute({ message }) {
+		const errorText = `Error: ${this.config.emojis.rabbit || '🐰'} No bunnies found.`;
+        try {
+			const utils = this.utils;
+			const responses = [
+				{ search: 'Looking for a bunnies...', found: 'Found one!' },
+			];
+
+			const response = responses[utils.getRandomInt(0, responses.length - 1)];
+			const msg = await this.sendMessage(message.channel, response.search);
+
+			let res = await superagent.get('https://api.bunnies.io/v2/loop/random/?media=poster');

```

Comment:
I think it would be better to group `this.sendMessage()` and `superagent.get()` in a `Promise.all([])` so we're not waiting for the message to send before querying the API.

Reply:
How will that be formatted? I've yet to write a Promise.all statement,

Reply:
Here's an example: https://github.com/FlexLabs/Dyno-fun/blob/1abf8af8e329de4d607ddac89f172a7732e5c578/src/commands/Space.js#L19